### ISSUE TITLE
Sharing profile_page endpoint's request

### ISSUE THREAD

**Issue by safaaltunel** [bounswe/2021SpringGroup10#217]
I'm having trouble with sending POST request to the "profile_page" endpoint.
Here is the code snippet that I'm using:
```javascript
const headers = {
    'Content-Type': 'application/json',
    'Access-Control-Allow-Origin': '*',
    'Access-Control-Allow-Methods': 'DELETE, GET, OPTIONS, POST, PUT',
    'Access-Control-Allow-Headers': 'Authorization, Content-Type, Access-Control-Allow-Headers, X-Requested-With, Access-Control-Allow-Origin'
}

const base_url = "https://cz2qlmf16e.execute-api.us-east-2.amazonaws.com/dev/api/"

const request_json = {
            "user_name": "Aayysy123"
            // 'profile_photo': photo,
            // 'bio': bio,
            // 'first_name': first_name,
            // 'last_name': last_name,
            // 'birth_date': birth_date
        };

        Axios({
            headers: headers,
            method: "POST",
            url: base_url + 'profile_page/',
            data: request_json,
        }).then(response => {
            
            setLoading(false)
            navigate('/home')
        }).catch(error => {
            
            setLoading(false)
            if (error.response) {
                // The request was made and the server responded with a status code
                // that falls out of the range of 2xx
                console.log(error.response.data);
                console.log(error.response.status);
                console.log(error.response.headers);
            } else if (error.request) {
                // The request was made but no response was received
                // `error.request` is an instance of XMLHttpRequest in the browser and an instance of
                // http.ClientRequest in node.js
                console.log(error.request);
            } else {
                // Something happened in setting up the request that triggered an Error
                console.log('Error', error.message);
            }
```
This function call always ends up within the body of "else  if(error.request)" statement. I really cannot find anything useful when I log error.request. Here is the log of error.request:
![response](https://user-images.githubusercontent.com/56367453/146679297-8621d1d6-2d81-4d25-bc43-9ec979dac1e5.png)

So can you please share with me at least one programmatic example of working request and response. I need to see exact parameters passed to the request(i.e. headers, body). 

This endpoint works fine with postman but postman does not share all the header parameters. Some of the parameters are calculated when request is sent. So I need

**Comment by OnurSefa** [comment]
When we inspected the returning values with @safaaltunel, we see that this is because the given endpoint is not hosted on AWS API Gateway. And it results with CORS policy error. This problem will be resolved after #196 handled.

**Comment by OnurSefa** [comment]
This issue is out of date and handled in different aspects. So that, i close this.
